presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -398,3 +398,35 @@ Hive Connector Limitations
 --------------------------
 
 :doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire partitions.
+
+
+Custom Storage Handlers
```

Comment:
Do we support this?  If not, we probably shouldn't document it.  Though it's useful as internal documentation.